article/app/views/fragments/recipeArticleBody.scala.html

```diff
@@ -25,10 +25,10 @@
 
     <div class="recipes__main">
         <div class="recipe__images-wrapper">
-            <div class="recipe__label">
-                <h2>food &amp; drink</h2>
-            </div>
-            <div class="recipe__image__wrapper recipe__image__wrapper--is-displayed">
+            <a href="https://www.theguardian.com/tone/recipes?CMP=recipe-article">
```

Comment:
@blongden73 given this is on www.theguardian.com, it would be better to do this:
```
@LinkTo("/tone/recipes?CMP=recipe-article")
```

Just like @SiAdcock was saying. That way on code, it will point to the code url and in dev it will be relative